features/list_dishes_on_landing_page.feature

```diff
@@ -23,3 +23,4 @@ Feature: List dishes on landing page
     And I would like to see "Great Apple Pie"
     And I would like to see "Cheese Cake" with a price of "60" kr
     And I would like to see "Great Cheese Cake"
+    Then show me the page
```

Comment:
Plese remove this. No debugger steps are needed when you are done with the feature

Reply:
Done